src/storage/page_buffer.c

```diff
@@ -8326,12 +8326,23 @@ pgbuf_request_data_page_from_page_server (const VPID * vpid, log_lsa target_repl
 
   size += cublog::lsa_utils::get_packed_size (pac, size);
   size += vpid_utils::get_packed_size (pac, size);
+  size += pac.get_packed_int_size (size);
```

Comment:
Function `pgbuf_request_data_page_from_page_server` is used in two contexts in function `pgbuf_read_page_from_file_or_page_server`. One of the contexts is used for comparison in the ATS-LS working scenarios and needs adaptation.